common.py

```diff
@@ -127,7 +124,7 @@ def generate_courtyard(
     dy = format_float(max_y + excess_y)
     return [
         '(polygon {} (layer {})'.format(uuid, 'top_courtyard'),
-        ' (width {}) (fill false) (grab_area false)'.format(COURTYARD_LINE_WIDTH),
+        ' (width {}) (fill false) (grab_area false)'.format(0.0),
```

Comment:
This changes the `width`. Was this intended?

Reply:
Yes this is intended because LibrePCB converts all courtyards to a line width of 0.0 during the file format upgrade. So when we generate v1.0 files, we need to use 0.0 as well to avoid reverting the migration.